pkgs/development/python-modules/dparse2/default.nix

```diff
@@ -29,14 +30,10 @@ buildPythonPackage rec {
   ];
 
   checkInputs = [
+    pipenv
     pytestCheckHook
   ];
 
-  disabledTestPaths = [
-    # Requries pipenv
```

Comment:
Everything seems to pass Just Fine without pipenv. I suspect this is an out of date comment.